owners/test/ownership/parser.test.js

```diff
@@ -594,7 +607,7 @@ describe('owners parser', () => {
 
         it('does not produce a reviewer set rule', () => {
           const {result} = parser.parseOwnersFileDefinition('OWNERS', fileDef);
-          expect(result.length).toEqual(0);
+          expect(result.length).toEqual(1);
```

Comment:
This seems like an odd change

Reply:
The test data used to have an empty list of rules (out of laziness) which was allowed, but now the validator yells if the set of rules or owners doesn't have at least one entry, so the test data now has 1 rule.